pkg/devfile/adapters/kubernetes/component/utils.go

```diff
@@ -0,0 +1,38 @@
+package component
```

Comment:
All the functions in `utils.go` can be reused by other adapters as well. Currently, only kubernetes adapter is implemented. But, if we need to another adapter, these functions can be re-used there as well.

So would it make sense to move these to a common location?

Reply:
One of the things I wanted to avoid was a large utils file but if you think it will likely be consumed by other adapters I'm fine moving it to a common place. It's common but still specific to kubernetes so I moved it to a new package `pkg/devfile/adapters/kubernetes/utils`